app/assets/stylesheets/govuk_publishing_components/components/govspeak/_button.scss

```diff
@@ -24,4 +24,33 @@
   .govuk-button--start {
     @extend .govuk-button--start;
   }
+
+  // stylelint-disable declaration-no-important
+  @include govuk-media-query($media-type: print) {
+    .gem-c-button {
+      display: inline-block;
+      box-shadow: none;
+      border: 2pt solid $govuk-print-text-colour;
+
+      &,
+      & * {
+        background-color: unset;
+        color: $govuk-print-text-colour !important;
+      }
+
+      &::before {
+        display: none;
```

Comment:
What's the before element and why does it need to be hidden?

Reply:
It seems like some kind of overlay for capturing full width clicks. It's definitely not needed for print, so I've deleted it. Good spot, thanks.

Reply:
Thanks. But it doesn't seem like the before element has any visual styles, so what's the need to hide it on print?

Reply:
Every little helps: The browser still has to compute to appearance of the element before composing the printout, so by hiding it we save it some time.

Reply:
I think that would apply if it was for everyone's benefit, but since everyone downloads and only a few people print I'd rather save everyone the few bytes of download than some people the rendering benefit when printing.